lib/KBox/src/KMessage.h

```diff
@@ -191,6 +196,94 @@ class IMUMessage: public KMessage {
     };
 };
 
+class NAVMessage: public KMessage {
+  private:
+    bool apMode, apHeadingMode, apWaypointMode, apDodgeMode;
+    double currentHeading, targetHeading, courseToWaypoint;
```

Comment:
I can guarantee from my experience with autopilots that have a dodgemode that just changes the targetheading but maintains control of the steering would be a bad idea.  Some autopilots have a "powersteering" mode that behaves similar to how you describe press -10/+10.  I would not use an autopilot that did not give me the option to temporarily deactivate the steering commands.  

And yes, just maintaining a set heading was my initial intention for version 1.  I thought it might he helpful to others and a hint to ourselves to leave some breadcrumbs in the code for future intended features like getting a heading to a waypoint, but that does not seem to be your preference.  The name of the page is immaterial.